docs/04-resources/08-runes/airdrop/deploy.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 3
+title: Deploying the Airdrop Machine Smart Contract Using Remix IDE
+sidebar_label: Deploy
+tags: [rsk, rootstock, resources, tutorials, runes, nft, Ethereum, dApps, smart contracts, airdrop]
+description: "This guide will walk you through deploying the Airdrop Machine smart contract using Remix IDE. The Airdrop Machine is designed to distribute tokens across multiple addresses, supporting both standard and Merkle tree-based airdrops. 
+"
+---
+This section will walk you through deploying the Airdrop Machine smart contract using Remix IDE. The Airdrop Machine is designed to distribute tokens across multiple addresses, supporting both standard and Merkle tree-based airdrops. 
```

Comment:
Merkle tree was explained right? Link to the explanation

Reply:
Using the word `we`, means we are there with the readers which is not possible (which means we are deploying alongside them).
This is a first person pronoun we should generally be avoided  because its an unclear reference and readers dont know who `we` refers to.

Reply:
Second person pronoun `You` is preferred or left the way it is as it is not wrong.